Cutelyst/action.cpp

```diff
@@ -207,6 +210,139 @@ bool Action::doExecute(Context *c)
         c->setState(ret);
         return ret;
     }
+
+#else
+
+    /*
+     * Qt 6.5 introduced a new variadic version of QMetaMethod::invoke() that
+     * does not work with our current implementation above. The following code
+     * is a fast fix to use the new version but there might be better / more elegant
+     * approaches.
+     *
+     * See: https://codereview.qt-project.org/c/qt/qtbase/+/422745
+     *
+     * TODO: check for more flexible implementation
+     */
+
+    if (d->evaluateBool) {
+        bool methodRet;
+
+        if (d->listSignature) {
+
+            QStringList args = c->request()->args();
+
+            ret = d->method.invoke(d->controller,
+                                   Qt::DirectConnection,
+                                   qReturnArg(methodRet),
+                                   c,
+                                   args);
+        } else {
+            QStringList args = c->request()->args();
+            // Fill the missing arguments
+            args.append(d->emptyArgs);
```

Comment:
remove this

Reply:
Ok

Reply:
@Danxomme No, not you. :) This was meant for me.